docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,359 @@
+
+By default, the Datagrid panel will not allow a user to edit data, but only show data like a standard visualization. To make full use of this new panel editing functionality you will need to enable the “enableDatagridEditing” feature toggle.
+But for now the Datagrid Panel will support the following features in G10:
+
+- Creating and deleting rows and columns
+- Data and column header edit or delete
+- Search functionality
+- Column freezing
+- Grid selection actions (copy/paste/delete)
+- Draggable columns and rows
+- Series select when pulling data from a datasource
+
+To learn more about the Datagrid panel, please refer to the official documentation.
+
+The list of features is already growing and of course we will look into those. So stay tuned for more features coming in the near future that will give an even better experience using the datagrid.
+
+### Drag and Drop
+
+<!-- Oscar Kilhed -->
+
+Finally connecting your local data is made easier! Welcome drag & drop in Grafana!
+The drag & drop functionality allows users to connect their csv, excel or numbers files by simply dragging and dropping them on a dashboard, or even into the editor directly.
+
+Once dropped a table will be created by default containing the data that is living in the file itself.
+
+By default, the drag & drop feature is not available out of the box. In order to utilize this functionality panel you will need to enable the `featuretogglename` feature toggle.
+
+As of now drag & drop supports the following scenarios in G10:
+
+- Drag & drop files on an empty dashboard
+- Drag & drop files into the panel editor
+- Replace files in the panel editor
+- Default table panel creation
+
+As of now the data is being stored on the dashboard json and has a 1MB size limit. Obviously it’s our mission to increase the limit without decreasing the performance in the near future.
+To learn more about drag & drop, please refer to the official documentation.
+
+The list of requests is already growing and of course we will look into those. So stay tuned for more features coming in the near future that will give an even better experience using the the drag & drop datasource.
+
+### Canvas GA
+
+<!-- Nathan Marrs -->
+
+We are promoting the canvas panel out of beta and into general availability. Over the past several months we have introduced several enhancements to the panel such as the ability to draw connections between elements, the ability to set the color and size of connections based on data, and the ability to add data links. We are excited to include canvas as a first class citizen in Grafana’s core panel library. To learn more about the Canvas panel, refer to [Canvas]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-canvas-service-graph.png" max-width="750px" caption="Canvas service graph" >}}
+
+### New Trend Panel
+
+<!-- Nathan Marrs -->
+
+_Available in **beta** in all editions of Grafana_
```

Comment:
@nmarrs we should use the term preview, not beta. cc @imatwawana See https://docs.google.com/spreadsheets/d/19GMKK09PAFZL6G5V54RpPZrnnTLGOuREVZVhk6q6Zjk/edit#gid=776746502 for details.